Cognite.Simulator.Utils/SimulationScheduler.cs

```diff
@@ -225,14 +225,15 @@ private async Task RunJob(ScheduledJob<V> job, CancellationToken mainToken)
             while (!mainToken.IsCancellationRequested || !job.TokenSource.Token.IsCancellationRequested)
             {
                 var routineRev = job.RoutineRevision;
-                var nextOccurrence = job.Schedule.GetNextOccurrence(DateTime.Now);
-                var delay = nextOccurrence - DateTime.Now;
+                var now = DateTime.UtcNow;
```

Comment:
Nitpick: Is it better to create a util function `DateTime.UtcNow` instead of using the same logic at different places?

Reply:
Encapsulated this into a separate function now, but yeah, I agree, we might want to consider a separate wrapper around DateTime